pkg/operator/ceph/object/spec.go

```diff
@@ -113,7 +113,7 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 			controller.DaemonVolumes(c.DataPathMap, rgwConfig.ResourceName),
 			c.mimeTypesVolume(),
 		),
-		HostNetwork:        c.clusterSpec.Network.IsHost(),
+		HostNetwork:        c.store.Spec.Network.IsHost(),
```

Comment:
I suspect `mds` also using the same network as cephcluster https://github.com/rook/rook/blob/0cd57e0b3d52ddad891095ad8a8340b9e4b48b24/pkg/operator/ceph/file/mds/spec.go#L63 
DO we need to update that too?

Reply:
I'd suggest let's just make this change for rgw

Reply:
If the object store network settings are not set, let's fall back to the cluster network settings